### DISCUSSION TITLE
Where do I put login info?

### DISCUSSION THREAD

**Discussion by predator290** [SecKatie/ha-wyzeapi#178] [Q&A]
Maybe im being dumb, but I never get a prompt or anywhere to put my login info for this. I'm new to HA and trying to learn. Can anyone help me out?

**Comment by SecKatie** [comment]
In the integrations panel as described in the readme when you add the Wyze Home Assistant Integration it will ask for your credentials.

**Reply by ggarman** [reply]
I'm having the same issue.  I add Wyze Home Assistant Integration via HACS and it never prompts, I just get the card in the Integrations tab.  If I click on it, it takes me to the readme and the dot menu has nothing useful in it.  I have uninstalled and reinstalled multiple times.  I'm running the current HA on a Pi.

@predator290 did you get yours to work?  If so, do tell.

Thanks!

**Reply by SecKatie** [reply]
Go to the integrations panel in the main Home Assistant settings, click add, and then choose Wyze Home Assistant Integration

**Reply by ancjsd** [reply]
I'm having the same issue. Tried what you suggested, but Wyze is not available from the main integrations panel in HA.  It's only availalble in the HACS intagrations panel.  And when you click on  it there after downloading it, it just takes me to the readme.  Thanks in advance for taking the time to respond!
--Jon
